developer/reference/runtime/register-component.mdx

```diff
@@ -146,6 +149,58 @@ runtime.registerComponent(Cube, {
 });
 ```
 
+### Adding a description
+
+This example shows how to add an `Circle` component with a rich description. We
+use markdown formatting for our tooltip descriptions.
+
+```tsx makeswift/components.tsx
+import { Style } from "@makeswift/runtime/controls";
+
+import { runtime } from "@/makeswift/runtime";
+
+function Circle({ className }) {
+  return <div className={className}>I'm a circle!</div>;
+}
+
+runtime.registerComponent(Circle, {
+  type: "circle-tester",
+  label: "Custom / Circle",
+  icon: "chats",
+  hidden: false,
+  description: `
+# This is a larger heading
+
+This is some text at the top of here
```

Comment:
Can we move the markdown into it's own variable to make this code snippet for `registerComponent` a bit more readable?